messages/snapshot.create.json

```diff
@@ -1,13 +1,15 @@
 {
-  "description": "snapshot a scratch org",
+  "description": "create a snapshot of a scratch org\nA snapshot is a point-in-time export of a scratch org. The export is stored in Salesforce and referenced by its unique name in a scratch org definition file.\n\nUse \"sfdx force:org:snapshot:get\" to get details, including status, about a snapshot creation request.\n\nTo create a scratch org from a snapshot, include the \"snapshot\" entry (instead of \"edition\") in the scratch org definition file and set it to the name of the snapshot. Then use \"sfdx force:org:create\" to create the scratch org.",
```

Comment:
```suggestion
  "description": "create a snapshot of a scratch org\nA snapshot is a point-in-time copy of a scratch org. The copy is stored in Salesforce and referenced by its unique name in a scratch org definition file.\n\nUse \"sfdx force:org:snapshot:get\" to get details, including status, about a snapshot creation request.\n\nTo create a scratch org from a snapshot, include the \"snapshot\" entry (instead of \"edition\") in the scratch org definition file and set it to the name of the snapshot. Then use \"sfdx force:org:create\" to create the scratch org.",
```